Makefile

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-BUILD_ARGS = --ssh=default
+BUILD_ARGS =
```

Comment:
This is being fixed in https://github.com/instructlab/instructlab/pull/988, it'd be great to align :)

Reply:
yep, this makes sense now too since the repos are open :)

Reply:
yep this makes sense too since the repos are open :)